### ISSUE TITLE
LIMIT OFFSET in delete()

### ISSUE THREAD

**Issue by lenyapugachev** [stephencelis/SQLite.swift#657]
How to enable SQLITE_ENABLE_UPDATE_DELETE_LIMIT, because this one
```swift
let messages = Table("messages")
.filter(Expression<String>("chatId") == chat[Expression<String>("id")])
.order(Expression<Date>("createdAt").desc)
.limit(Int.max, offset: index > 29 ? 1 : 30)

try db.run(messages.delete())
```
removes ALL rows from `messages`

**Comment by Dridus** [comment]
@jberkel since #722 is now merged, should this be closed?

**Comment by jberkel** [comment]
let's leave it open for now, might take a bit before the release branch gets merged into master

**Comment by jberkel** [comment]
Released as part of 0.11.4
